src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -50,23 +50,6 @@ public void UsesApplicationNameAsControllerNameIfNoDeclaringType()
             Assert.Equal(nameof(EndpointMetadataApiDescriptionProviderTest), apiDescription.ActionDescriptor.RouteValues["controller"]);
         }
 
-        [Fact]
```

Comment:
@davidfowl @halter73 and @pranavkm , I think this test is not needed anymore since we are adding the default content type (```application/json```) as ```IAcceptsMetadata``` when we call the ```RequestDelegate``` Create method. Any thoughts ?

Reply:
Why isn't this required? Shouldn't we test it anyways?

Reply:
This is fine, understood.